pegasus/sites.v3/code.org/views/music/music_lab_carousel_videos.haml

```diff
@@ -0,0 +1,35 @@
+:ruby
+  videos = [
+    {
+      code: "GqlP5s7KTl4",
+      download_path: "//videos.code.org/music_lab/Music_Lab_Make_Music_Learn_Code.mp4",
+      caption: hoc_s("music_lab.make_music_learn_code"),
+    },
+    {
+      code: "oZhFQ1yjWhY",
+      download_path: "//videos.code.org/music_lab/Music_Lab_Aloe_Blacc.mp4",
+      caption: hoc_s("music_lab.artist.singer_songwriter", markdown: :inline, locals: {artist_name: "Aloe Blacc"}),
+    },
+    {
+      code: "Fysf1rD-RX0",
+      download_path: "//videos.code.org/music_lab/Music_Lab_Rosa_Linn.mp4",
+      caption: hoc_s("music_lab.artist.singer_songwriter", markdown: :inline, locals: {artist_name: "Rosa Linn"}),
+    },
+    {
+      code: "1TDma4wg3dE",
+      download_path: "//videos.code.org/music_lab/Music_Lab_Drew_Taggart_from_The_Chainsmokers.mp4",
+      caption: "Drew Taggart from The Chainsmokers",
```

Comment:
should this be translated?

Reply:
Here's a screenshot with the updated string working:
<img width="310" alt="Screenshot 2024-05-15 at 12 25 13 PM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/9edd5a70-9349-41dc-b14f-a404e5ab9359">